src/components/link/link.tsx

```diff
@@ -68,21 +68,21 @@ const EuiLink = forwardRef<HTMLAnchorElement | HTMLButtonElement, EuiLinkProps>(
       rel,
       type = 'button',
       onClick,
+      disabled,
       ...rest
     },
     ref
   ) => {
-    const classes = classNames(
-      'euiLink',
-      colorsToClassNameMap[color],
-      className
-    );
-
     if (href === undefined) {
       const buttonProps = {
-        className: classes,
+        className: classNames(
+          'euiLink',
+          disabled ? 'euilink-disabled' : colorsToClassNameMap[color],
```

Comment:
Spelling error here and BEM naming conventions requires the use of double dashes to indicate a state.
```suggestion
          disabled ? 'euiLink--disabled' : colorsToClassNameMap[color],
```

Reply:
Ugh sorry, the BEM **is** correct, but the spelling error is still there.